package.json

```diff
@@ -29,7 +30,8 @@
     "validate-translations": "node scripts/validate_translations.js",
     "generate_licenses": "yarn licenses generate-disclaimer > src/assets/disclaimer.txt && yarn licenses list --json > src/assets/licenses.json",
     "generate_licenses_list": "yarn licenses list > licenses.txt",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "ui": "cd $(git rev-parse --show-toplevel) && yarn run --silent ts-node ./scripts/ui/ui.ts"
```

Comment:
shouldn't check-environment be used here as well?

Reply:
Wanted to give an update. I'm working on this help text issue right now. Commander.js documentation seems to indicate that adding `command.addToHelp('after', 'My help string')` should do what we want but that is not working. Seems our commander.js version is wildly out of date. I've updated it to latest (4.3 -> 10.0.1) and there is a lot of breakage. Next commit should have the help text updates